runtime/frame_descriptors.c

```diff
@@ -111,19 +105,8 @@ static void fill_hashtable(
   }
 }
 
-/* [add_frame_descriptors(table, frametables, new_frametables)]
```

Comment:
Any specific reason for the removal of this comment?

Reply:
The comment is explaining things that are non-obvious because the function has a complex API. (It takes a table and two lists, that are related to each other in a subtle way, and mutates all of three arguments in a subtle way). I felt that, once the function is simplified by the change in the commit you are looking at, then the things that needed explanations before are not necessary to explain anymore, so I can just get rid of the comment.